tests/features/start.feature

```diff
@@ -15,4 +15,13 @@ Feature: Start server
 
     Scenario: I can start server
         When I start server
-        Then I have no errors
\ No newline at end of file
+        Then I have no errors
+
+    Scenario: Server has "alive" state
+        When I get server state 
+        """
+        {
+          "State": "1"
```

Comment:
Could we use states' names instead of numbers? Numbers aren't good for readability.

Reply:
Yeah, I just tested it. Protogen generates map for enum, so it can parse state's names instead of numbers.
```
// Enum value maps for State.
var (
	State_name = map[int32]string{
		0: "Alive",
		1: "Stopped",
		2: "Dead",
	}
	State_value = map[string]int32{
		"Alive":   0,
		"Stopped": 1,
		"Dead":    2,
	}
)
```